shopfloor_mobile/static/wms/src/services/odoo.js

```diff
@@ -39,33 +39,29 @@ export class OdooMixin {
                 if (_.isUndefined(handler)) {
                     handler = this._handle_error;
                 }
-                return response.json().then(json => {
-                    return {error: handler.call(this, response, json)};
-                });
+                return {error: handler.call(this, response)};
             }
             return response.json();
         });
     }
-    _error_info(response, json) {
-        return _.extend({}, json, {
-            // strip html
-            description: $(json.description).text(),
```

Comment:
hmm this is required to properly display the error https://github.com/OCA/wms/blob/13.0/shopfloor_mobile/static/wms/src/scenario/mixins.js#L378

Reply:
But where does the `description` comes from in the response, in the backend ?

Reply:
The problem is https://github.com/OCA/wms/pull/145/files#diff-3b7d8d1af55526336a23050aa28b6605a0eb0625d5e129bde2cfd683d8ee42e5L42
crashes with 404 from Odoo (their body contains  html not json data.